### DISCUSSION TITLE
ArmourStand damage event

### DISCUSSION THREAD

**Discussion by zfms4188** [Roadhog360/Et-Futurum-Requiem#148] [Improvements]
Could you change the armor stand to drop after taking mortal damage? Is the current click time for detection?

Currently, I can't destroy the armor stand by preventing other players on the server, as the armor stand drops are not caused by injury

(Please contact me if there is any difficulty, I am using Google Translate.)

**Comment by Roadhog360** [comment]
The armor stands do not drop when broken in Creative mode.

Otherwise, I'm having a very hard time understanding what you actually mean if that is not what you meant.

**Reply by zfms4188** [reply]
XD...That's not what I meant
I've noticed that at the moment. Armor stands are not dropped as a result of being mortally wounded.
So I can't protect the armor stand on the server by preventing damage events from happening

**Reply by Roadhog360** [reply]
> XD...That's not what I meant I've noticed that at the moment. Armor stands are not dropped as a result of being mortally wounded. So I can't protect the armor stand on the server by preventing damage events from happening

So you don't get the armor stand back when you break it?

**Reply by zfms4188** [reply]
> > XD...That's not what I meant I've noticed that at the moment. Armor stands are not dropped as a result of being mortally wounded. So I can't protect the armor stand on the server by preventing damage events from happening
> 
> So you don't get the armor stand back when you break it?

Yes, I can.
However, the server has a lot of other players who might steal things from it

**Comment by Roadhog360** [comment]
I've just checked and interacting with armor stands already fires an event.
